pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -29,10 +29,19 @@ store:
     driverName: pgx # ENV: KUMA_STORE_POSTGRES_DRIVER_NAME
     # Connection Timeout to the DB in seconds
     connectionTimeout: 5 # ENV: KUMA_STORE_POSTGRES_CONNECTION_TIMEOUT
-    # Maximum number of open connections to the database
+    # MaxConnectionLifetime (applied only when driverName=pgx) is the duration since creation after which a connection will be automatically closed
+    maxConnectionLifetime: "1h" # ENV: KUMA_STORE_POSTGRES_MAX_CONNECTION_LIFETIME
+    # MaxConnectionLifetimeJitter (applied only when driverName=pgx) is the duration after maxConnectionLifetime to randomly decide to close a connection.
+    # This helps prevent all connections from being closed at the exact same time, starving the pool.
+    maxConnectionLifetimeJitter: "0s" # ENV: KUMA_STORE_POSTGRES_MAX_CONNECTION_LIFETIME_JITTER
```

Comment:
Why not put a non-zero parameter? Feels like it's the sort of params you set and forget about so having it non 0 would be good :) I guess 1m is a good default.

Reply:
👍 will change to "1m"